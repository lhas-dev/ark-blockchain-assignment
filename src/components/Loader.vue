<template>
  <img class="mx-auto" src="../assets/loader.svg" />
</template>

<script>
export default {};
</script>
