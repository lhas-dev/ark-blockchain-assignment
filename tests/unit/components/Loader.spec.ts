import { shallowMount } from "@vue/test-utils";
import Loader from "@/components/Loader.vue";

describe("Loader.vue", () => {
  it("renders correctly", () => {
    const wrapper = shallowMount(Loader);
    expect(wrapper).toMatchSnapshot();
  });
});
