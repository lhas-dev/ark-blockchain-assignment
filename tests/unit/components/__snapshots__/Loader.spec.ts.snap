// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader.vue renders correctly 1`] = `<img src="../assets/loader.svg" class="mx-auto">`;
